package.json

```
@@ -25,11 +25,15 @@
     "ready": "pnpm lint && pnpm test",
     "prepare": "husky install",
     "release": "npx nx run-many --target=build --skip-nx-cache && pnpm changeset publish",
+    "release:check": "changeset status --verbose --since=origin/develop",
```

```suggestion
    "release:status": "changeset status --verbose --since=origin/develop",
```
small nit will do in follow up pr